package.json

```diff
@@ -9,16 +9,32 @@
     "posttest": "npm run test:lint && npm run test:git-history"
   },
   "devDependencies": {
```

Comment:
What is the reason for moving all the `devDependencies` into the parent `package-lock.json`